scout/commands/load/case.py

```diff
@@ -64,11 +80,12 @@ def case(
     ped,
     update,
     config,
-    no_variants,
```

Comment:
I have never used this option, the organisation does not use it and it does nothing. Looks like it was a leftover from a dev process to implement a test for loading cases without variants? See
https://github.com/Clinical-Genomics/scout/commit/c5ee18c9bd431d2a8b488844470cf3a095090091

Reply:
I'm not surprised, we have quite a number of those!